main.py

```diff
@@ -316,20 +320,31 @@ def encrypt_payload_and_update_message_using_dks(dks_endpoint, message):
     return encrypt_payload(encryption_key, message)
 
 
+def get_escaped_json_string(json_string):
```

Comment:
incorrect var name, change to message or change message to Json_string

Reply:
Good spot! Fixed @mike-iw